docs/resources/netflow_exporter_policy.md

```diff
@@ -0,0 +1,153 @@
+  - `uni/tn-{name}/exporterpol-{name}`
+
+## GUI Information ##
+
+* Locations:
+  - `Tenants -> Policies -> NetFlow -> NetFlow Exporters`
+  - `Fabric -> Access Policies -> Policies -> Interface -> NetFlow -> NetFlow Exporters`
+
+## Example Usage ##
+
+The configuration snippet below creates a Netflow Exporter Policy with only required attributes.
+
+```hcl
+
+resource "aci_netflow_exporter_policy" "example_tenant" {
+  parent_dn = aci_tenant.example.id
+  name      = "netfow_exporter"
+}
+
+```
+The configuration snippet below shows all possible attributes of the Netflow Exporter Policy.
+
+!> This example might not be valid configuration and is only used to show all possible attributes.
+
+```hcl
+
+resource "aci_netflow_exporter_policy" "full_example_tenant" {
+  parent_dn      = aci_tenant.example.id
+  annotation     = "annotation"
+  description    = "description"
+  dscp           = "AF11"
+  dst_addr       = "2.2.2.1"
+  dst_port       = "https"
+  name           = "netfow_exporter"
+  name_alias     = "name_alias"
+  owner_key      = "owner_key"
+  owner_tag      = "owner_tag"
+  source_ip_type = "custom-src-ip"
+  src_addr       = "1.1.1.1/10"
+  ver            = "v9"
+  annotations = [
+    {
+      key   = "key_0"
+      value = "value_1"
+    }
+  ]
+  tags = [
+    {
+      key   = "key_0"
+      value = "value_1"
+    }
+  ]
+}
+
+```
+
+All examples for the Netflow Exporter Policy resource can be found in the [examples](https://github.com/CiscoDevNet/terraform-provider-aci/tree/master/examples/resources/aci_netflow_exporter_policy) folder.
+
+## Schema ##
+
+### Required ###
+
+* `parent_dn` - (string) The distinguished name (DN) of the parent object, possible resources:
+  - [aci_tenant](https://registry.terraform.io/providers/CiscoDevNet/aci/latest/docs/resources/tenant) ([fvTenant](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvTenant/overview))
+  - The distinguished name (DN) of classes below can be used but currently there is no available resource for it:
+    - [infraInfra](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/infraInfra/overview)
+
+* `name` (name) - (string) The name of the Netflow Exporter Policy object.
+
+### Read-Only ###
+
+* `id` - (string) The distinguished name (DN) of the Netflow Exporter Policy object.
+
+### Optional ###
+  
+* `annotation` (annotation) - (string) The annotation of the Netflow Exporter Policy object.
+  - Default: `orchestrator:terraform`
+* `description` (descr) - (string) The description of the Netflow Exporter Policy object.
+* `dscp` (dscp) - (string) IP dscp value.
+  - Default: `CS2`
+  - Valid Values: `AF11`, `AF12`, `AF13`, `AF21`, `AF22`, `AF23`, `AF31`, `AF32`, `AF33`, `AF41`, `AF42`, `AF43`, `CS0`, `CS1`, `CS2`, `CS3`, `CS4`, `CS5`, `CS6`, `CS7`, `EF`, `VA`.
+* `dst_addr` (dstAddr) - (string) Remote node destination IP address.
+* `dst_port` (dstPort) - (string) Remote node destination port.
```

Comment:
Can only the string representation of a port be provided or also a number like http or 80?